suzieq/poller/sq_poller.py

```diff
@@ -116,11 +116,14 @@ def controller_main():
     parser.add_argument(
         '--run-once',
         type=str,
-        choices=['gather', 'process'],
-        help=('Collect the data from the sources and terminates. gather store '
-              'the output as it has been collected, process performs some '
-              'processing on the data. Both cases store the results in a '
-              'plain output file, one for each service.')
+        choices=['gather', 'process', 'debug'],
+        help=('''The poller do not run forever, three modes are available:
+        (1) gather: store the output as it has been collected,
+        (2) process: performs some processing on the data.
+        Both cases store the results in a plain output file,
+        one for each service, and exit.
+        (3) debug: build the device list and exit without polling
+        the nodes.''')
```

Comment:
As I said earlier, why not use --debug? This is more of a question, not a request to change the way you're doing it.